src/components/remotecontrol/remotecontrol.js

```diff
@@ -331,7 +379,7 @@ define(["browser", "datetime", "backdrop", "libraryBrowser", "listView", "imageL
         function updatePlayPauseState(isPaused, isActive) {
             var context = dlg;
             var btnPlayPause = context.querySelector(".btnPlayPause");
-            btnPlayPause.querySelector("i").innerHTML = isPaused ? "&#xE037;" : "pause";
+            btnPlayPause.querySelector("i").innerHTML = isPaused ? "play_circle_filled" : "pause_circle_filled";
```

Comment:
Use the UTF-8 codes (&#x12AB) or convert this to use classes instead.

Icons with underscores don't work on WebOS when using ligatures like this, due to a bug in the browser on that platform. We're generally moving everything to use classes, as the UTF-8 codes are hard to read and don't work if the user disables custom fonts.